plugins/woocommerce-blocks/tests/e2e/tests/product-category/product-category.block_theme.spec.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { expect, test } from '@woocommerce/e2e-playwright-utils';
+
+const blockData = {
+	name: 'Products by Category',
+	slug: 'woocommerce/product-category',
+};
+
+test.describe( `${ blockData.slug } Block`, () => {
+	test( 'can be inserted in Post Editor and it is visible on the frontend', async ( {
+		editorUtils,
+		editor,
+		admin,
+		frontendUtils,
+	} ) => {
+		await admin.createNewPost();
+		await editor.insertBlock( { name: blockData.slug } );
+		const blockLocator = await editorUtils.getBlockByName( blockData.slug );
+		await expect( blockLocator.getByRole( 'listitem' ) ).toHaveCount( 9 );
```

Comment:
Shouldn't we select a category in this test similar to what's done in `product-tag`? Otherwise only the "Select category" screen is shown.

Reply:
Yep! I addressed with [cceae5e](https://github.com/woocommerce/woocommerce/pull/46671/commits/cceae5e40ed8971782d3d6edd5888934a33a7fe0)